packages/yourloops/pages/login/login-page.tsx

```diff
@@ -25,101 +25,223 @@
-      margin: `${theme.spacing(2)} 0`
-    },
-    cardActions: {
-      justifyContent: 'center'
+const LOGO_COLOR_LIGHT = '#40BAE9'
+const styles = makeStyles({ name: 'login-page-styles' })((theme: Theme) => ({
+  appBar: {
+    borderBottom: `1px solid ${theme.palette.divider}`,
+    zIndex: theme.zIndex.drawer + 1,
+    backgroundColor: theme.palette.common.white,
+    color: 'var(--text-base-color)',
+    paddingBlock: theme.spacing(2),
+    paddingInline: theme.spacing(7)
+  },
+  backgroundImage: {
+    position: 'absolute',
+    right: 0,
+    bottom: -1,
+    zIndex: -1
+  },
+  button: {
+    marginRight: theme.spacing(2),
+    paddingInline: theme.spacing(3),
+    textTransform: 'capitalize',
+    width: 136
+  },
+  heading: {
+    fontFamily: 'MuseoSlab',
+    fontSize: '1.5rem',
+    fontWeight: 'bold',
+    marginBottom: theme.spacing(2)
+  },
+  info: {
+    fontSize: '1.1rem',
+    lineHeight: theme.spacing(4)
+  },
+  infoContainer: {
+    color: 'var(--text-base-color-light)',
+    display: 'flex',
+    flexDirection: 'column',
+    height: '100%',
+    justifyContent: 'center',
+    paddingInline: theme.spacing(10),
+    width: '45%'
+  },
+  hoverable: {
+    fontWeight: 'bold',
+    color: 'var(--logo-color)',
+    fontSize: '1rem',
+    '&:hover': {
+      color: LOGO_COLOR_LIGHT
+    }
+  },
+  laptopImage: {
+    position: 'absolute',
+    width: 800,
+    right: '6%',
+    top: 'calc(50% - 250px)'
+  },
+  link: {
+    color: 'var(--logo-color)',
+    fontSize: '1.5rem',
+    fontWeight: 'bold',
+    marginTop: theme.spacing(2),
+    '&:hover': {
+      color: LOGO_COLOR_LIGHT
+    }
+  },
+  loginButton: {
+    backgroundColor: 'var(--logo-color)',
+    '&:hover': {
+      backgroundColor: LOGO_COLOR_LIGHT
+    }
+  },
+  registerButton: {
+    backgroundColor: 'var(--text-base-color-light)',
+    '&:hover': {
+      backgroundColor: '#2e2e2d'
     }
   }
-})
+}))
 
 const LoginPage: FunctionComponent = () => {
-  const { loginWithRedirect, error, logout } = useAuth0()
-  const { t } = useTranslation('yourloops')
-  const { classes: { cardContent, card, cardActions } } = loginStyle()
+  const { loginWithRedirect, error } = useAuth0()
+  const { t, i18n } = useTranslation()
+  const history = useHistory()
+  const alert = useAlert()
+  const { classes, theme, cx } = styles()
+  const auth0EmailVerifyErrorMessage = 'Please verify your email before logging in.'
 
-  const onClickLogout = async (): Promise<void> => {
-    await logout({ returnTo: window.location.origin })
-  }
+  const redirectToSignup = async (): Promise<void> => await loginWithRedirect({ screen_hint: 'signup' })
 
-  if (error) {
-    console.log(error)
-  }
+  useEffect(() => {
+    if (error) {
+      if (error.message === auth0EmailVerifyErrorMessage) {
+        history.replace('/verify-email')
+        return
+      }
+      alert.error(error.message)
+    }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+  }, [error])
```

Comment:
I'm curious, why didn't you add alert and history in the deps?

Reply:
If I add alert in deps it makes an infinite loop... Once any alert method is called, it re-render the component, the error is still here so it triggers the alert... and again and again and again... thanks to the useEffect !

Reply:
ahahaha
And history?

Reply:
#balek